ENIGMAsystem/SHELL/Platforms/SDL/Android/android.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Platforms/General/PFdevice.h"
+#include <SDL_android.h>
+
+namespace enigma_user
+{
+
+float device_get_tilt_x()
+{ 
+  float Values[3],tilt_x;
+  Android_JNI_GetAccelerometerValues(Values);
+  tilt_x = Values[0];
+  return tilt_x;
+}
+
+float device_get_tilt_y()
+{
+  float Values[3],tilt_y;
+  Android_JNI_GetAccelerometerValues(Values);
+  tilt_y = Values[1];
+  return tilt_y;
+}
+
+float device_get_tilt_z()
+{
+  float Values[3],tilt_z;
+  Android_JNI_GetAccelerometerValues(Values);

```

Comment:
If the tilt is actually slow, then it may need to be cached like we do with window caption. Alternatively, we could just poll it once every tick from a callback in the main loop. You'd store that array in namespace `enigma` and have these functions return `enigma::tilt_values[...]` instead.